tests/account_manager.rs

```diff
@@ -86,14 +86,12 @@ async fn different_seed() -> Result<()> {
         .await?;
 
     // Generating a new account needs to return an error, because the seed from the secret_manager is different
-    assert!(
-        manager
-            .create_account()
-            .with_alias("Bob".to_string())
-            .finish()
-            .await
-            .is_err()
-    );
+    assert!(manager
+        .create_account()
+        .with_alias("Bob".to_string())
+        .finish()
+        .await
+        .is_err());
```

Comment:
@pelumi-ajayi for some reason your fmt changed it so it's failing here now

Reply:
Not sure what's going on either. I'll have a look